### DISCUSSION TITLE
Welcome to usgscsm Discussions!

### DISCUSSION THREAD

**Discussion by jlaura** [DOI-USGS/usgscsm#354] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by oleg-alexandrov** [comment]
It appears that one should click on the "(un)watch" button in the upper-right, select "custom", and ensure the "discussions" checkbox is selected in order to receive notifications of discussions.
